daras_ai_v2/base.py

```diff
@@ -876,6 +876,14 @@ def _get_default_pr_title(self):
         recipe_title = self.get_root_pr().title or self.title
         return f"{self.request.user.first_name_possesive()} {recipe_title}"
 
+    def _get_default_pr_visibility(self, workspace: Workspace | None = None):
+        if not workspace:
+            workspace = self.current_workspace
```

Comment:
I don't think we should use a different workspace here than the published run one

Reply:
perhaps move this logic inside create_with_version?

Reply:
+ make the visibility kwarg optional, so that add_version() retains the same visibility, whereas create_with_version() uses the default